@xen-orchestra/backups/RemoteAdapter.js

```diff
@@ -576,20 +576,30 @@ class RemoteAdapter {
     return stream
   }
 
-  async readDeltaVmBackup(metadata) {
+  async readDeltaVmBackup(metadata, settings) {
```

Comment:
I've lost the comment I wrote :disappointed: 

I think it would be best to pass a `ignoredVdis` set instead, which would be created in the calling code:

```js
const ignoredVdis = new Set(
  Object.entries(mapVdisSrs)
    .filter(([_, srUuid]) => srUuid === null)
    .map(([vdiUuid]) => vdiUuid)
)
```